lib/chef/client.rb

```diff
@@ -241,7 +241,8 @@ def load_node
     # Chef::Node:: The updated node object
     def build_node
       policy_builder.build_node
-      @run_status = Chef::RunStatus.new(node, events)
+      @run_status.node = node
+      @run_status.events = events
```

Comment:
the events object should be wired up correctly in the initializer already i think.